src/features/dropIntoEditor.ts

```diff
@@ -0,0 +1,81 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path'
+import * as vscode from 'vscode'
+import * as URI from 'vscode-uri'
+
+const imageFileExtensions = new Set<string>([
+  '.bmp',
+  '.gif',
+  '.ico',
+  '.jpe',
+  '.jpeg',
+  '.jpg',
+  '.png',
+  '.psd',
```

Comment:
I don't think an HTML page is capable of displaying a preview of a PSD file (but I might be wrong). If that's the case then I would remove this extension.

Reply:
I will try

Reply:
you were right. no PSD...